spec/features/user_can_update_glossary_entries_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'sign_up_helper'
+require 'create_glossary_entry_helper'
+
+feature 'Glossary edit' do
+  context 'user can edit glossary term' do
+    let(:user) { build(:user) }
+    let(:glossary) { build(:glossary) }
```

Comment:
Again, if we are not testing creation through the UI I would create at this stage...